mkt/feed/tests/test_views.py

```diff
@@ -1374,6 +1400,39 @@ def test_shelf_top(self):
         eq_(data['objects'][0]['item_type'],
             feed.FEED_TYPE_SHELF)
 
+    def test_websites(self):
+        self.feed_factory()
+        self.featured_mow_factory(n_row=6, n_www=5)
+        res, data = self._get()
+        tags = [mow['keywords'][0] for mow in data['websites']]
+        eq_(len(data['websites']), 11)
+        eq_(tags.count('featured-website-restofworld'), 6)
+        eq_(tags.count('featured-website'), 5)
+
+    def test_websites_prefer_regional(self):
```

Comment:
have a test where half the websites are regional and then the bottom half is global